map/src/util/GzipBase64.mjs

```diff
@@ -24,7 +24,9 @@ export async function compressFromString(str) {
 }
 
 export async function decompressToJSON(str) {
-    return JSON.parse(await decompressFromBase64(str));
+    if (str) {
+        return JSON.parse(await decompressFromBase64(str));
+    }
```

Comment:
Please add default return value: null or {} (as you wish depends on how you use the result)